### ISSUE TITLE
repeating 'read ECONNRESET' errors on skype bridge working without new events

### ISSUE THREAD

**Issue by MurzNN** [matrix-hacks/matrix-puppet-skype#12]
When matrix-puppet-skype works - it periodically repeats this error in room:
```
{ poll: An error happened while processing the polled messages
  caused by Error: read ECONNRESET
    at exports._errnoException (util.js:1024:11)
    at TLSWrap.onread (net.js:610:25)
  name: 'poll',
  [_message]: 'An error happened while processing the polled messages',
  data: {},
  cause: 
   { Error: read ECONNRESET
       at exports._errnoException (util.js:1024:11)
       at TLSWrap.onread (net.js:610:25)
     [stack]: 'Error: read ECONNRESET\n    at exports._errnoException (util.js:1024:11)\n    at TLSWrap.onread (net.js:610:25)',
     [message]: 'read ECONNRESET',
     code: 'ECONNRESET',
     errno: 'ECONNRESET',
     syscall: 'read' },
  [_stack]: 'poll: An error happened while processing the polled messages\n  caused by Error: read ECONNRESET\n    at exports._errnoException (util.js:1024:11)\n    at TLSWrap.onread (net.js:610:25)',
  [_stackContainer]: undefined,
  message: 'poll: An error happened while processing the polled messages\n  caused by Error: read ECONNRESET\n    at exports._errnoException (util.js:1024:11)\n    at TLSWrap.onread (net.js:610:25)' } ﻿
```
What does this error mean and how to solve it?

**Comment by Sorunome** [comment]
it is an error within the skype-http library where it doesn't reconnect correctly on polling errors

**Comment by MurzNN** [comment]
Can we successfully ignore them, or better is to restart puppet after this error?

**Comment by Sorunome** [comment]
we would need to have the skype client reconnect. no need to restart the entire puppet

Am 16. August 2017 08:01:30 MESZ schrieb Alexey Murz Korepov <notifications@github.com>:

**Comment by MurzNN** [comment]
But current matrix-puppet-skype version do this reconnect automatically? If not, how I can force reconnect to Skype without puppet restart?

**Comment by Sorunome** [comment]
it currently does not reconnect automatically. You would have to add that code in....

Am 16. August 2017 09:02:48 MESZ schrieb Alexey Murz Korepov <notifications@github.com>:

**Comment by nr23730** [comment]
Solved by #28
